src/packages/@teams/features/team-invite/layout/TeamInvite.svelte

```diff
@@ -0,0 +1,445 @@
+<script lang="ts">
+  export let handleModalState: (flag: boolean) => void;
+  export let teamName: string = "";
+  export let teamId: string = "";
+  import closeIcon from "$lib/assets/close.svg";
+  import {
+    base64ToURL,
+    createDynamicComponents,
+    validateEmail,
+  } from "$lib/utils/helpers";
+  import { notifications } from "@library/ui/toast/Toast";
+
+  import { TeamRole, WorkspaceRole } from "$lib/utils/enums/team.enum";
+  import Dropdown from "$lib/components/dropdown/Dropdown.svelte";
```

Comment:
I think we are not using any component from deprecate?

Reply:
removed